.travis.yml

```diff
@@ -10,10 +10,8 @@ python:
   - "pypy"
 sudo: required
 group: deprecated-2017Q2
-services:
-  - docker
 before_install:
-  - docker run -d -p 10101:10101 pilosa/pilosa:v0.6.0
+  - curl -o pilosa https://storage.googleapis.com/pilosa/pilosa-sumreduce && chmod +x pilosa && ./pilosa server &
```

Comment:
@yuce did you mean to leave this reference to googleapis in here?

Reply:
Yes, that downloads a custom built Pilosa server, since we didn't have a release with the latest updates yet.

Reply:
@codysoyland let's discuss the release plan for range-encoded fields and try to apply this at the same time so that we don't have to reference an external dependency like googleapis.

Reply:
@travisturner I've uploaded the binary to google cloud since it was convenient. Do we have somewhere else like s3 to put these kinds of files? We will definitely need to have out-of-band builds for custom built binaries in the future, since otherwise we can't run integration tests on CI for client code which needs features in not-yet-released pilosa servers.